src/main/java/software/amazon/event/kafkaconnector/mapping/DefaultEventBridgeMapper.java

```diff
@@ -30,10 +30,20 @@ public class DefaultEventBridgeMapper implements EventBridgeMapper {
   private final EventBridgeSinkConfig config;
   private final JsonConverter jsonConverter = new JsonConverter();
   private final ObjectMapper objectMapper = new ObjectMapper();
+  private final DetailTypeMapper detailTypeMapper;
 
   public DefaultEventBridgeMapper(EventBridgeSinkConfig config) {
     jsonConverter.configure(singletonMap("schemas.enable", "false"), false);
     this.config = config;
+    try {
+      var myClass = Class.forName(config.detailTypeMapperClass);
+      var constructor = myClass.getDeclaredConstructor();
+      this.detailTypeMapper = (DetailTypeMapper) constructor.newInstance();
+      this.detailTypeMapper.configure(config);
+    } catch (Exception e) {
+      // This will already be verified in the Config Validator
+      throw new RuntimeException("Topic to Detail-Type Mapper Class can't be loaded.");
```

Comment:
Yup, should throw earlier. Was wondering whether we throw ConnectException or RuntimeE here? What are we doing in other classes?

Reply:
This is already validated in validateDetailTypeMapperClass in the EventBridgeSinkConfigValidator which throws a ConfigException. However, we still have to handle the checked exception here.